<launch>
	<node pkg="final_race" type="city_driving.py" name="city_driving" output="screen">
		<param name="pub_topic" value="/odom"/>
        <param name="sub_topic" value="/odom"/>
	</node>
</launch>
